### ISSUE TITLE
md-tabs rtl support

### ISSUE THREAD

**Issue by leonzinger** [angular/material#1078]
When i add md-tabs on rtl page the tabs ink bar always underlines the opposite side

**Comment by ThomasBurleson** [comment]
@robertmesserle - any thoughts?

**Comment by matthiasg** [comment]
the tabs highlight correctly in 1.0.0-rc6 at least but they arent right aligned and they should reverse order (at least by default), just like the rest of flex items does

**Comment by HosseinMoradiDavijani** [comment]
This solved my issue:
```css
md-tabs[dir="rtl"] {
    .md-tab {
        float: right;
    }
    md-tab-content {
        &.md-left:not(.md-active) {
            -webkit-transform: translateX(100%);
            transform: translateX(100%);
        }
        &.md-right:not(.md-active) {
            -webkit-transform: translateX(-100%);
            transform: translateX(-100%);
        }
    }
    md-tabs-canvas {
        .md-dummy-wrapper {
            left: auto;
            right: 0;
        }

        md-pagination-wrapper {
            left: auto;
            right: 0;
        }
    }
}
```
and:
```html
<md-tabs dir="rtl">
.
.
.
</md-tabs>
```

**Comment by EladBezalel** [comment]
@HosseinMoradiDavijani it seems like a PR material, we will really appreciate it if you would!

**Comment by DevVersion** [comment]
Seems like this is fixed with SHA 08bf4f93763d2aea9901d697f8cc3786d242725f

**Comment by anilk533** [comment]
Ltr direction for material tabs not working.
here is the issue.
https://codepen.io/djidel/pen/NpMpmY

please let me know how to solve

thank you
